controllers/test-data-controllers.js

```diff
@@ -13,4 +13,20 @@ exports.getTopics = (req, res, next) => {
 
 exports.getInfo = (req, res, next) => {
     res.status(200).send(jsonFile);
-  };
\ No newline at end of file
+};
+
+exports.getByArticleId = (req, res, next) => {
+    const { article_id } = req.params
+    console.log(req.params, "controllers params");
+    selectArticleById(article_id)
+    .then((article)=>{
+        res.status(200).json({ article })
+    })
+    .catch((err)=>{
+      if (err.msg === "Not Found") {
```

Comment:
In our catch block, all we should be doing is passing the error to our error handling middleware functions to return the error, we shouldn't be returning the error here.